src/components/Person/Person.jsx

```diff
@@ -1 +1,23 @@
-// export const Person = ({ person }) => ();
+const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${person.name}`}
+    </h2>
+    {person.age && (
+      <p className="Person__age">
+        {`I am ${person.age}`}
+      </p>
+    )}
+    {person.isMarried ? (
+      <p className="Person__partner">
+        {`${person.partnerName} is my ${
+          person.sex !== 'm' ? 'husband' : 'wife'
+        }`}
```

Comment:
It's difficult to read. Create variable for this value `person.sex !== 'm' ? 'husband' : 'wife'`